src/test/java/com/verisign/storm/metrics/GraphiteMetricsConsumerTest.java

```diff
@@ -151,57 +157,42 @@ public void shouldReadDataAndSendLong(Collection<DataPoint> dataPoints) {
 
     // Then the reporter should send properly formatted metric messages to Graphite
     HashMap<String, Double> expMap = buildExpectedMetricMap(dataPoints);
-    verify(consumer).appendToBuffer(getExpectedMetricPrefix(), expMap, taskInfo.timestamp);
+    verify(consumer).appendToReporterBuffer(getExpectedMetricPrefix(), expMap, taskInfo.timestamp);
   }
 
-  @Test(dependsOnMethods = { "shouldInitializeGraphiteReporter" }, dataProvider = "generateDataPointsFloat")
-  public void shouldReadDataAndSendFloat(Collection<DataPoint> dataPoints) {
-    //Given a consumer
-    GraphiteMetricsConsumer consumer = spy(new GraphiteMetricsConsumer());
+  @DataProvider(name = "generateMapDataPoints") public Object[][] generateMapDataPoints() {
+    int numDataPoints = 5;
+    int numDataPointValues = 5;
 
-    Map stormConf = ImmutableMap
-        .of("metrics.graphite.host", testStormSrcWorkerHost, "metrics.graphite.port", testStormSrcWorkerPort.toString(),
```

Comment:
a constant or variable for all of these strings will make your tests more dynamic